packages/iTwinUI-react/src/core/Header/HeaderSplitButton.tsx

```diff
@@ -54,19 +52,9 @@ export const HeaderSplitButton: HeaderSplitButtonComponent = React.forwardRef(
         title={title}
         ref={ref}
       >
-        <button
-          className='iui-header-breadcrumb-button'
-          onClick={onClick}
-          ref={forwardedRef}
-          type={type}
-          {...rest}
-        >
-          {startIcon &&
-            React.cloneElement(startIcon, {
-              className: startIcon.props.className,
-            })}
+        <HeaderBasicButton onClick={onClick} ref={forwardedRef} {...rest}>
```

Comment:
You aren't including `startIcon` anymore (or `endIcon`). Why is that?

Reply:
The `startIcon` isn't being destructed in `HeaderSplitButton`, so it is passed to be destructed in `HeaderBasicButton` through `...rest`

The `endIcon` for the button is being passed through `...rest` as well. The caret icon in `HeaderSplitButton` is handled in the `<button>` element that takes care of the split part of the `HeaderSplitButton`

I also just realized that `onClick` isn't being handled in `HeaderSplitButton` either, so I've updated the code to have it pass through `...rest` as well